### PR TITLE
improve PHP Hint for Expectation

### PR DESCRIPTION
**Pull request by sohelzerdoumi**

### CODE REVIEW

File: src/Expectation.php [hunk 1]
```diff
@@ -14,28 +14,28 @@
...
- * @method Expectation toBeCloseTo(float $expected, int $precision) passes if actual is close to expected in some precision
- * @method Expectation toBeGreaterThan(mixed $expected) passes if actual if greater than expected
- * @method Expectation toBeLessThan(mixed $expected) passes if actual is less than expected
- * @method Expectation toThrow(mixed $expected = null) passes if actual throws the expected exception
- * @method Expectation toMatch(string $expected) passes if actual matches the expected regexp
- * @method Expectation toEcho(string $expected) passes if actual echoes the expected string
- * @method Expectation toMatchEcho(string $expected) passes if actual echoes matches the expected string
- * @method Expectation toReceive(string $expected) passes if the expected method as been called on actual
- * @method Expectation toReceiveNext(string $expected) passes if the expected method as been called on actual after some other method
+ * @method \Kahlan\Matcher\ToBe toBe(mixed $expected) passes if actual === expected
```
Comment (samsonasik) [inline src/Expectation.php]: as already in `Kahlan` namespace, it should be doable with `Matcher\ToBe` instead ?

Comment (jails) [reply src/Expectation.php]: Good catch, let's do that :+1:

**Comment by samsonasik** [comment]
@sohelzerdoumi merged to `dev` branch, I've made a [change](https://github.com/kahlan/kahlan/commit/bd41be894cad30e5fb7377b6f17f4e4f2ffafd24) during merge

**Comment by samsonasik** [comment]
@sohelzerdoumi Thank you ;)

**Comment by sohelzerdoumi** [comment]
thx ;)

**Pull request merged**